src/module.ts

```diff
@@ -6,11 +6,17 @@ import { joinURL } from 'ufo'
 
 export interface ModuleOptions {
   /**
-   * Directus API URL
-   * @default process.env.NUXT_DIRECTUS_URL
+   * Directus API URL, customizable at runtime via NUXT_PUBLIC_DIRECTUS_URL environment variable.
+   * @default 'https://localhost:8055'
```

Comment:
Default should be the env variable

Reply:
IIRC Nuxt overrides any default variable when they follow the runtime config structure (or at least I'm 100% sure with an empty string).
So for serverside configs any `NUXT_MYMODULE_*` would override any `runtimeConfig.mymodule.*` defaults. Likewise for public configs any `NUXT_PUBLIC_MYMODULE_*` would override any defaults for `runtimeConfig.public.mymodule.*`

But yes, I didn't realize that I should default to an empty string, so that we could throw an error notifing the developer that they didn't specify the Directus url.